packages/server/lib/refreshTokens.ts

```diff
@@ -28,15 +28,23 @@ export function refreshTokens(): void {
 }
 
 export async function exec(): Promise<void> {
+    const startTime = Date.now();
+    const maxDuration = 23 * 60 * 60 * 1000;
+
     logger.info('[refreshTokens] starting');
 
-    const staleConnections = await connectionService.getOldConnections({ days: 7, limit: 500 });
+    const staleConnections = await connectionService.getOldConnections({ days: 1 });
```

Comment:
can we still have a limit (even if very high)? Just to make sure any refresh token problem don't cascade into making this call returns a huge number of connections

Reply:
Point taken about not returning a huge number of connections. I think I'd rather chunk this actually, will update this tomorrow. Thanks